codeclimate-govet.go

```diff
@@ -35,12 +50,14 @@ func main() {
 				pieces := strings.Split(line, ":")
 
 				if len(pieces) < 3 {
-					// engine.PrintWarning()
+					fmt.Fprintf(os.Stderr, "Unexpected format for the following output: %v\n", out)
+					fmt.Fprintf(os.Stderr, "\nError: %v\n", err)
```

Comment:
Still not sure we want this line. Where does err come from? I think it's the same err from L30, which we would've already printed